development-guidelines.md

```diff
@@ -69,6 +69,106 @@ Examples:
 * Use [italia-utils](https://github.com/teamdigitale/italia-utils) for generating `io-ts` models from OpenAPI specs.
 * Refer to the [React Typescript cheatsheet](https://github.com/typescript-cheatsheets/react-typescript-cheatsheet) to type React Components
 
+### Coding: some practical examples
```

Comment:
since this may grow too much to be embedded into this doc (which has a broader high level scope); I'd rather create a `development-stylguide.md` which contains all the code samples

Reply:
moreover I'd add a section title to group by topics (ie. io-ts / fp-ts, React, express and controllers, functions..)

Reply:
Yes of course. Moreover, we could add an index in _typescript_ section where all links are listed